plugins/lando/lando.plugin.zsh

```diff
@@ -0,0 +1,54 @@
+LANDO_ZSH_SITES_DIRECTORY="$HOME/Sites"
+
+LANDO_ZSH_CONFIG_FILE=./.lando.yml
+
+# Enable multiple commands with lando.
+function  artisan \
+          composer \
+          drush \
+          gulp \
+          npm \
+          wp \
+          yarn {
+
+  if checkForFile ; then
+    lando "$0" "$@"
+  else
+    command "$0" "$@"
+  fi
+}
+
+# Check for the file in the current and parent directories.
+checkForFile(){
+
+  local current_directory="$PWD"
+
+  # Bash is backwards. 0 is true 1 (non-zero) is false.
+  # Only bother checking for lando within the Sites directory.
+  if [[ "$PWD" == "$LANDO_ZSH_SITES_DIRECTORY"* ]]; then
+
+    # "Checking for file: $LANDO_ZSH_CONFIG_FILE within $LANDO_ZSH_SITES_DIRECTORY..."
+
+    while true; do
```

Comment:
This probably should be `[[ "$current_directory" != / ]]` instead of true. Otherwise if code isn't tight there may be an infinite loop.